tests/cases/fourslash/codeFixChangeJSDocSyntax15.ts

```diff
@@ -1,10 +1,9 @@
 /// <reference path='fourslash.ts' />
 //// var f = <[|function(number?): number|]>(x => x);
 
-// note: without --strict, number? --> number, not number | null
 verify.codeFix({
     description: "Change 'function(number?): number' to '(arg0: number) => number'",
     errorCode: 8020,
     index: 0,
-    newRangeContent: "(arg0: number) => number",
+    newRangeContent: "(arg0: number | null) => number",
```

Comment:
Given the original was `function(number?): number` shouldnt this be `(arg0?: number) => number`
i guess seeing `null` is bad as `undefined` is what people without --strict would think of?

Reply:
jsdoc `?` means `| null`, jsdoc `=` means `| undefined` (affecting arity if in trailing argument). I know, confusing, but we get it right internally. It didn't include the `| null` before because rather than using the existing node tree, it was using the semantic type, which _didn't_ include `null` because `strictNullChecks` wasn't on. This is pretty much strictly an improvement, as we now retain more information from the original annotation.